disk/smart-monitoring.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
```

Comment:
copyright

Reply:
Could you please be a bit clearer what you want me to do here?
It is acutally like this in every script e.g.: https://github.com/nextcloud/vm/blob/98f3ac340cfeab813133c6397d90e61e8d874a0c/static/change-ncadmin-profile.sh#L3